### PR TITLE
Fix the issue from Pe-dry run

### PR DESCRIPTION
**Pull request by lancehaha**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -188,9 +188,10 @@ Format: `find n/NAME p/PHONE r/ROOM_NUMBER t/TAG`
   e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
 * The order of the `TAGS` does not matter e.g. `t/friends t/colleagues` works the same
   as `t/colleagues t/friends`
-* `TAG` search is case-sensitive e.g. `Friends` will not match `friends`
-* When searching by `TAG`, the contact must match every `TAG` in the input fields to appear in the listed results.
-* If the user inputs multiple fields for the `find`, then only contacts that match all the input fields will appear in the listed results.
+* Tag search is case-insensitive, `Friends` will match `friends`
+* In tag search, the contacts have to contain all the specified TAGs in order to match
```
Comment (chashaobao123) [inline docs/UserGuide.md:192]: remember to add full stops for lines 191 and 192

File: docs/UserGuide.md [hunk 2]
```diff
@@ -188,9 +188,10 @@ Format: `find n/NAME p/PHONE r/ROOM_NUMBER t/TAG`
...
+* If the user inputs multiple fields in parameter, then only the contacts that match all the field
+in the parameter will be listed out.
```
Comment (SeanFoongjt) [inline docs/UserGuide.md:194]: Should parameter be better defined for non technical users?

File: docs/UserGuide.md (hunk 1 above)
Comment (chashaobao123) [reply docs/UserGuide.md:192]: Perhaps we can rephrase line 192 to something more intuitively understood? 

"The results of tag search will only contain the contacts that match all of the specified tags.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -188,9 +188,10 @@ Format: `find n/NAME p/PHONE r/ROOM_NUMBER t/TAG`
   e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
 * The order of the `TAGS` does not matter e.g. `t/friends t/colleagues` works the same
   as `t/colleagues t/friends`
-* `TAG` search is case-sensitive e.g. `Friends` will not match `friends`
-* When searching by `TAG`, the contact must match every `TAG` in the input fields to appear in the listed results.
-* If the user inputs multiple fields for the `find`, then only contacts that match all the input fields will appear in the listed results.
+* Tag search is case-insensitive, `Friends` will match `friends`
```
Comment (SeanFoongjt) [inline docs/UserGuide.md:191]: Is tag search not case sensitive?

File: docs/UserGuide.md (hunk 2 above)
Comment (chashaobao123) [inline docs/UserGuide.md:194]: Maybe can rephrase to 

"If multiple parameters, `NAME`, `PHONE`, `ROOM_NUMBER`, and/or `TAG`, are specified, the results of the search will only contain contacts that match all the specified parameters.

**Review by EdaYeo — COMMENTED** [review]
LGTM, just 2 nits!

File: docs/UserGuide.md [hunk 4]
```diff
@@ -212,7 +212,7 @@ Format: `delete INDEX`
 * There is a confirmation popout after the user enters the command but before the delete is actually done.
   click `OK` to continue or `Cancel` to abort the deletion.
 
-![delete popout](images/deleteConfirmation.png)
+![delete popup](images/Delete popup.png)
```
Comment (EdaYeo) [inline docs/UserGuide.md:215]: Would it be better to add the image as Delete popup screenshot to standardise the screenshot size, changing the "Delete popup.png" filename to "Deletepopup.png"?

**Review by EdaYeo — COMMENTED** [review]
LGTM, just one last detail about screenshot size standardisation!

**Review by EdaYeo — APPROVED** [review]
LGTM!

**Pull request merged**